src/hooks/useContract.ts

```diff
@@ -391,3 +400,70 @@ export function useLiquidityGaugeContract(
 ): LiquidityGaugeV5 | null {
   return useContract(gaugeAddress, LIQUIDITY_V5_GAUGE_ABI) as LiquidityGaugeV5
 }
+
+// This section instantiate new contract without Hooks
+export function getGaugeContract(
+  library: Web3Provider,
+  chainId: ChainId,
+  address: string,
+  account: string,
+): LiquidityGaugeV5 | ChildGauge {
+  if (chainId === ChainId.MAINNET) {
+    return getContract(
+      address,
+      LIQUIDITY_V5_GAUGE_ABI,
+      library,
+      account,
+    ) as LiquidityGaugeV5
+  }
+
+  return getContract(address, CHILD_GAUGE_ABI, library, account) as ChildGauge
+}
+
+export function getGaugeMinterContract(
+  chainId: ChainId,
+  account: string,
+  library: Web3Provider,
+) {
+  return getContract(
+    GAUGE_MINTER_ADDRESSES[chainId],
+    GAUGE_MINTER_ABI,
+    library,
+    account,
+  ) as Minter
+}
+
+export function getChildGaugeFactory(library: Web3Provider, address: string) {
+  if (!address) return null
+  return getContract(
+    address,
+    CHILD_GAUGE_FACTORY_ABI,
+    library,
+  ) as ChildGaugeFactory
+}
+
+export const getVotingEscrowContract = (
+  chainId: ChainId,
+  account: string,
+  library: Web3Provider,
+): VotingEscrow => {
+  return getContract(
+    VOTING_ESCROW_CONTRACT_ADDRESS[chainId],
+    VOTING_ESCROW_CONTRACT_ABI,
+    library,
+    account,
+  ) as VotingEscrow
+}
+
+export const getChildOracle = (
+  chainId: ChainId,
+  account: string,
+  library: Web3Provider,
+): ChildOracle => {
+  return getContract(
```

Comment:
I'll keep this in mind as I go through the refactor. It might not even be necessary to have this check because if we wrap everything in a try/catch, we can have the `GaugeProvider` return null for gauges if any errors come up. 

I'm leaning towards letting `getContract` throw an error, and have the providers handle what and when to return null.